src/server/JasmineGraphInstanceService.cpp

```diff
@@ -80,4301 +144,5243 @@ void *instanceservicesession(void *dummyPt) {
-            }
+                if (sockfd < 0) {
+                    instance_logger.log("Cannot create socket", "error");
+                    return 0;
+                }
 
-            write(connFd, JasmineGraphInstanceProtocol::OK.c_str(), JasmineGraphInstanceProtocol::OK.size());
-            instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::OK, "info");
+                if (host.find('@') != std::string::npos) {
+                    host = Utils::split(host, '@')[1];
+                }
 
-            JasmineGraphHashMapLocalStore graphDB;
-            JasmineGraphHashMapCentralStore centralDB;
+                server = gethostbyname(host.c_str());
+                if (server == NULL) {
+                    instance_logger.log("ERROR, no host named ", "error");
+                    return 0;
+                }
 
-            std::map<std::string, JasmineGraphHashMapLocalStore> graphDBMapLocalStoresPgrnk;
-            if (JasmineGraphInstanceService::isGraphDBExists(graphID, partitionID)) {
-                JasmineGraphInstanceService::loadLocalStore(graphID, partitionID, graphDBMapLocalStoresPgrnk);
-            }
+                bzero((char *)&serv_addr, sizeof(serv_addr));
+                serv_addr.sin_family = AF_INET;
+                bcopy((char *)server->h_addr, (char *)&serv_addr.sin_addr.s_addr, server->h_length);
+                serv_addr.sin_port = htons(port);
+                if (Utils::connect_wrapper(sockfd, (struct sockaddr *)&serv_addr, sizeof(serv_addr)) < 0) {
+                    instance_logger.log("ERROR connecting", "error");
+                    // TODO::exit
+                    return 0;
+                }
 
-            if (JasmineGraphInstanceService::isInstanceCentralStoreExists(graphID, partitionID)) {
-                JasmineGraphInstanceService::loadInstanceCentralStore(graphID, partitionID, graphDBMapCentralStores);
-            }
+                bzero(data, INSTANCE_DATA_LENGTH + 1);
+                int result_wr = write(sockfd, JasmineGraphInstanceProtocol::HANDSHAKE.c_str(),
+                                      JasmineGraphInstanceProtocol::HANDSHAKE.size());
 
-            graphDB = graphDBMapLocalStoresPgrnk[graphID + "_" + partitionID];
-            centralDB = graphDBMapCentralStores[graphID + "_centralstore_" + partitionID];
-
-            map<long, map<long, unordered_set<long>>> egonetMap =
-                calculateLocalEgoNet(graphID, partitionID, serverPort, graphDB, centralDB, workerSockets);
-
-            string instanceDataFolderLocation =
-                Utils::getJasmineGraphProperty("org.jasminegraph.server.instance.datafolder");
-            string attributeFilePart = instanceDataFolderLocation + "/" + graphID + "_egonet_" + partitionID;
-            ofstream partfile;
-            partfile.open(attributeFilePart, std::fstream::trunc);
-            for (map<long, map<long, unordered_set<long>>>::iterator it = egonetMap.begin(); it != egonetMap.end();
```

Comment:
Have we moved the egonet related logic somewhere or we have simply deleted it?

Reply:
It's now in `static void worker_egonet_command()` function

Reply:
Ok